new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js

```diff
@@ -17,6 +17,8 @@ import styles from './FiatBalanceAlerts.styles.js'
 const useStyles = makeStyles(styles)
 
 const NAME = 'fiatBalanceAlerts'
+// const CASH_IN_KEY = 'fiatBalanceAlertsCashIn'
```

Comment:
Remove annotations from file (unless it's temporary?)

Reply:
Yes, it is all temporary. Stuff in this PR which is commented is to be uncommented later. Resolving all conversations related to this.